lib/Elements/Element.tsx

```
@@ -0,0 +1,373 @@
+      case "tr":
+        return (
+          <Slot props={{ token, children }} render={streamdown.snippets.tr}>
+            <View style={[streamdown.theme.tr.base, { flexDirection: "row" }]}>
+              {children}
+            </View>
+          </Slot>
+        );
+
+      case "td":
+        const tdToken = token as any;
+        if (tdToken.rowspan > 0) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)